enclave_manager/Dockerfile

```diff
@@ -161,7 +161,7 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/*
 
 # Install setuptools packages using pip because
-RUN pip3 install setuptools
+RUN pip3 install setuptools jsonschema
```

Comment:
Is it required here as well??

Reply:
Since i moved all the validation with common Python will check and remove if this is still required. Thanks for the catch

Reply:
removed